test/data/modules/dependent_includes/tasks/DependentIncludes.cpp

```diff
@@ -0,0 +1,14 @@
+#include "DependentIncludes.hpp"
+#include <cross_consumer/TaskStates.hpp>
```

Comment:
Failing line without adaptions in project.rb

Reply:
Thanks. The codegen test suite has always been crap. Nowadays we are using the rock build tests suite which is a lot more flexible.

For fail-to-build errors, just create the necessary orogen project(s) under `build_tests/orogen/` and add them to the buildconf.